templates/include/footer.html

```diff
@@ -65,7 +65,8 @@ <h6 class="ds-h4">Blue Button 2.0 API</h6>
           <li><a href="https://bluebutton.cms.gov/developers">Documentation</a></li>
           <li><a href="https://bluebutton.cms.gov/support">Support</a></li>
           <li><a href="https://bluebutton.cms.gov/resources">Resources</a></li>
-          <li><a href="https://sandbox.bluebutton.cms.gov/docs/openapi">API Docs</a></li>
+          <li><a href="https://sandbox.bluebutton.cms.gov/docs/openapi">API Reference</a></li>
```

Comment:
This could be a relative URL.

```suggestion
          <li><a href="/docs/openapi">API Reference</a></li>
```